### ISSUE TITLE
A11y_Azure Data Studio_Nov2023_Database_Database Properties Preview_Resize: After applying resize complete name and left navigation of Database Properties Preview dialog is not visible.

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/azuredatastudio#25124]
**"Please do not close this bug. This bug should only be closed by Trusted Tester after verification"**
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.
## Git hub Tags:
#A11yTCS; #A11yMAS; #A11ySev1; #DesktopApp; #WCAG1.4.4; #FTP; #SH_Azure Data Studio_Nov2023_Win32_Dec2023; #SQL Azure Data Studio; #Win32; #Resize; #Benchmark;
## Environment Details:
Application Name: Azure Data Studio
Operating System: Win11
Azure data studio 1.47.0
### Prerequisites to apply resize
1. Change system resolution to 1280*768 and drag text size to 200%
## Repro Steps:
1. Install Azure data studio application and open it
2. Make sure to enable "Preview" feature in settings.
3. Tab till "Database" and press enter to expand the tree.
4. Tab till any database and pressF10 to open context menu.
5. Tab till "Properties Preview" and press enter.
6. Apply resize
7. You will observe that after applying resize complete name and left navigation of Database Properties Preview dialog is not visible.
## Actual Result:
After applying resize complete name and left navigation of Database Properties Preview dialog is not visible
## Expected Result:
After applying resize complete name and left navigation of Database Properties Preview dialog is not visible
## User Impact:
Low visibility user who uses application in resize will not be able to see complete information.
### Attachment:

https://github.com/microsoft/azuredatastudio/assets/124254981/b14151d3-175c-4abd-ac3e-020e3ae1ba50

**Comment by corivera** [comment]
@kapilvaishna This has been fixed in the latest insiders build by adding a horizontal scrollbar to the dialog. You can find the latest windows build of ADS here: https://azuredatastudio-update.azurewebsites.net/latest/win32-x64-user/insider

**Comment by kisantia** [comment]
@kapilvaishna please verify that this is fixed in the latest insiders build: https://azuredatastudio-update.azurewebsites.net/latest/win32-x64-user/insider

**Comment by v-jagansai** [comment]
@kisantia Verified the issue , Issue is fixed , Hence clsoing the bug 
Version: 1.49.0-insider (user setup)
Commit: 99bc417adcb300cb3ab6b32da9a407
Attachment 
https://github.com/microsoft/azuredatastudio/assets/109065315/826a2cb0-8001-49ac-a53d-f4ca9cd7a87a

1bbb66c100
Date: 2024-02-28T06:30:09.585Z
VS Code: 1.82.0
Electron: 25.9.8
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.22631
#closed; #Regressed:02-28-24;

**Comment by kisantia** [comment]
closing as this was verified fixed above.
